app/javascript/leaflet/viewers/esri/feature_layer.js

```diff
@@ -0,0 +1,106 @@
+import { featureLayer } from "esri-leaflet";
```

Comment:
This is related to https://github.com/geoblacklight/geoblacklight/pull/1392/files#diff-fee17ec305e10d4416e6dfa4e2b2ad6223d976b376ee0bf6c1d0fbf4c447a1e2R12 but for some reason the http://localhost:3000/catalog/f406332e63eb4478a9560ad86ae90327_18 is not lading markers correctly and it thinks they are http://localhost:3000/catalog/../marker-icon.png which isn't anything.

Reply:
I see now, marker-icon was in vendor/assets/images/marker-icon.png. Don't know why my github search didn't surface that. We are going to need to figure out where to put them.